Cargo.toml

```diff
@@ -11,3 +11,10 @@ opt-level = 3
 [profile.bench]
 debug = 1
 opt-level = 3
+
+[profile.test]
+opt-level = 0
+lto = false
+debug-assertions = true
+codegen-units = 1
+panic = 'unwind'
```

Comment:
Minor nitpick: should be a newline at the end of this file.

Reply:
I will revert these changes before merging. These profiles were added only to test if setting codegen-units to 1 would improve either compiletimes or remove some issues of incremental compilation.